source/getting-started/build-host-setup.rst

```diff
@@ -48,12 +49,14 @@ Please ensure to install **Dekstop Development with C++** along with Microsoft V
 
 Install the **exact** versions (if specified) of the following tools to the designated directories:
 
-* Python 3.6 - **C:\\Python36**
-* NASM 2.12.02 - **C:\\Nasm**
+* Python 3.8.10 - **C:\\Python38**
+* NASM 2.16.01 - **C:\\Nasm**

```

Comment:
Do we need to remove NASM and IASL from line 38 - $ sudo apt-get install -y build-essential iasl python uuid-dev nasm openssl gcc-multilib qemu git

Reply:
thanks for pointing that out, I have removed them to avoid overwritten issue.